app/src/main/java/com/nutomic/syncthingandroid/service/SyncthingRunnable.java

```diff
@@ -70,6 +70,11 @@ public SyncthingRunnable(Context context, Command command) {
         mContext = context;
         mSyncthingBinary = Constants.getSyncthingBinary(mContext);
         mLogFile = Constants.getLogFile(mContext);
+
+        // Get preferences relevant for meaningful logs in bug reports.
+        Log.v(TAG, "SyncthingRunnable: syncthing-core option restartOnWakeup = " + Boolean.toString(mPreferences.getBoolean("restartOnWakeup", false)));
```

Comment:
This seems like a fairly random place to wedge this in. Isn't there a better place?

Reply:
Agreed, it doesnt make sense to log app preferences, we can just ask users for them.

Or we could decide to log all app preferences, but that would have to be a seperate PR. But not even Syncthing does that.